### ISSUE TITLE
Subtest as first test with no plan

### ISSUE THREAD

**Issue by ajbrowe** [tupilabs/tap4j#15]
Further to issue #12 we have some tests that start with a subtest, but due to the a changeable nature of the thing we are testing we are not able to set a test plan up front so the first line in our TAP output is an indented subtest line.

An example of this TAP output is:

```
    ok 1 - subtest 1a
    ok 2 - subtest 1b
    1..2
ok 1 - Subtest 1
    ok 1 - subtest 2a
    ok 2 - subtest 2b
    1..2
ok 2 - Subtest 2
1..2
ok
```

a perl test that will produce this output is as follows

```
use Test::More;

subtest 'Subtest 1' => sub {
    ok(1,'subtest 1a');
    ok(1,'subtest 1b');
};
subtest 'Subtest 2' => sub {
    ok(1,'subtest 2a');
    ok(1,'subtest 2b');
};
done_testing();
```

Currently this produces an error such as:

```
org.tap4j.parser.ParserException: Error parsing TAP Stream: Invalid indentantion. Check your TAP Stream. Line: ok 1 - subtest 1a
```

A workaround for us is to stick an always true test before the first subtest but this is a bit of a PITA to be honest.

P.S. Also note this spelling of **indentantion** I think should probably be **indentation**

**Comment by japod** [comment]
I have just submitted a new pull request (https://github.com/tupilabs/tap4j/pull/40) that should address the issue.

**Comment by kinow** [comment]
:+1: thanks @japod ! Will try to review/merge it as soon as possible :-)

Thanks

**Comment by kinow** [comment]
Fixed by pull request #40. Thanks to @japod ! Preparing 4.2.0 tonight (NZ time). Should be available from Maven central in about 10 hours.
